symfony.plugin.zsh

```diff
@@ -5,6 +5,7 @@ alias sfnew='symfony new';
 alias sfstart='sf server:start';
 alias sfrun='sf server:run -vvv';
 alias sfstop='sf server:stop';
+alias sfstop='sf server:stop; sleep 0.5; sf server:start';
```

Comment:
@thatside, There is a typo here. The alias should have a name `sfrestart` or something like that.